open_academy/models/session.py

```diff
@@ -7,3 +7,6 @@ class Session(models.Model):
     start_date = fields.Date()
     duration = fields.Integer()
     number_of_seats = fields.Integer()
+    instructor_id = fields.Many2one('res.partner')
+    courses_id = fields.Many2one('course', required=True)
```

Comment:
I missed this field, a session can only have one course, so it should be singular

```suggestion
    course_id = fields.Many2one('course', required=True)
```